soccer/gameplay/plays/Defensive_Scramble/basic_defensive_scramble.py

```diff
@@ -0,0 +1,77 @@
+import main
+import robocup
+import behavior
+import constants
+import enum
+
+import standard_play
+import tactics.positions.submissive_goalie as submissive_goalie
+import tactics.positions.submissive_defender as submissive_defender
+import evaluation.opponent as eval_opp
+import tactics.positions.wing_defender as wing_defender
+import skills.mark as mark
+import skills.capture as capture
+import tactics.defense
+import situational_play_selection
+
+## Play that uses submissive defenders and wingers to defend
+#  an attack close to our goal and cause one robot to capture.
+#  
+#  By default, we will use standard defense (two submissive 
+#  defenders, one goalie) and two wing defense robots. The
+#  remaining robot will mark the highest threat robot. 
+class BasicDefensiveScramble(standard_play.StandardPlay):
+    
+    _situationList = [
+        situational_play_selection.SituationalPlaySelector.Situation.DEFENSIVE_SCRAMBLE
+    ] # yapf: disable
+
+
+    def __init__(self, num_defenders=2, num_wingers=2):
+        super().__init__(continuous=True)
+
+        self.num_defenders = num_defenders
+        self.num_wingers = num_wingers
+
+        self.add_transition(behavior.Behavior.State.start,
+                            behavior.Behavior.State.running, lambda: True,
+                            'Immediately')
+
+        self.remove_all_subbehaviors()
```

Comment:
This is due to the standard play defense stuff right?

Reply:
This is probably extra code. I'll make the changes